systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -113,6 +125,40 @@ public void testDeployUndeploy() {
         testDockerImagesForKafkaConnect();
     }
 
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectS2ICluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG_CONVERTER_SCHEMAS_DISABLED)
+    public void testDeployS2IWithMongoDBPlugin() {
```

Comment:
Do we want this in `ConnectClusterIT`, or should S2I functionality live in its own `ConnectS2IClusterIT` class?